### PR TITLE
added unit test for post feature

### PR DESCRIPTION
**Pull request by adrean-hipe**
### OVERVIEW

- Added unit testing for post feature
- Moved unit testing to Unit folder in test
- Used other seeders on the base seeder file
- Modified user factory to include use of username instead

### NOTES

- php artisan test

### CODE REVIEW

File: laravel-app/database/seeders/UserInformationSeeder.php [hunk 1]
```diff
@@ -29,9 +31,11 @@ public function run(): void
                 'middle_name' => 'Elanor',
                 'bio' => 'Hello World',
                 'gender' => 'male',
+                'created_at' => now(),
+                'updated_at' => now(),
             ],
         ];
 
-        DB::table('user_information')->insert($user_information);
+        UserInformation::insert($user_information);
```
Comment (hipe01mark) [inline laravel-app/database/seeders/UserInformationSeeder.php:39]: Is this working? AFAIK if you want to create data using eloquent it should be **create()** right?

File: laravel-app/database/seeders/UserPostsTableSeeder.php [hunk 2]
```diff
@@ -17,14 +17,16 @@ public function run(): void
...
             [
                 'user_id' => 1,
                 'content' => 'Time is gold, spend it wisely :)',
                 'created_at' => now(),
+                'updated_at' => now(),
             ],
         ];
 
-        DB::table('user_posts')->insert($user_posts);
+        UserPost::insert($user_posts);
```
Comment (hipe01mark) [inline laravel-app/database/seeders/UserPostsTableSeeder.php:30]: Is this working? AFAIK if you want to create data using eloquent it should be create() right?

Comment (adrean-hipe) [reply laravel-app/database/seeders/UserPostsTableSeeder.php:30]: Insert works sir

**Review by hipe01mark — APPROVED** [review]
LGTM 🙇

**Review by bpocrafael — APPROVED** [review]
LGTM 🚀

**Pull request merged**